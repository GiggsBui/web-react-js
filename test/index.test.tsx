import renderer from 'react-test-renderer';
import IndexPage from '../pages';

describe('Index page', () => {
    it('should match the snapshot', () => {
        const user = {
            name: 'Giggs',
            email: 'giggs@namtech.sg',
            phone: '84973637103',
            address: '165/16c Văn Thân, P8, Q6, TPHCM',
        };
        const tree = renderer.create(<IndexPage user={user} />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
