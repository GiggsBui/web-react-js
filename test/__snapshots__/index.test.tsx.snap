// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<main>
  <h1>
    Wellcome 
    Giggs
     join to Your Flower App
  </h1>
  <p>
    Email: 
    giggs@namtech.sg
  </p>
  <p>
    Phone: 
    84973637103
  </p>
  <p>
    Address: 
    165/16c Văn Thân, P8, Q6, TPHCM
  </p>
  <button
    onClick={[Function]}
  >
    Click me
  </button>
</main>
`;
